lib/spack/spack/cmd/clean.py

```diff
@@ -27,7 +28,7 @@
 class AllClean(argparse.Action):
     """Activates flags -s -d -f -m and -p simultaneously"""
     def __call__(self, parser, namespace, values, option_string=None):
-        parser.parse_args(['-sdfmpb'], namespace=namespace)
+        parser.parse_args(['-sdfmp'], namespace=namespace)
```

Comment:
We need to make clear in the help message that `--all` does not include bootstrap.

Reply:
Done in bb537c9